fhir-validation/pom.xml

```diff
@@ -18,6 +18,16 @@
             <artifactId>fhir-model</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>fhir-path</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>fhir-path</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
is the second one intended for the same scope?

Reply:
Removed unnecessary dependency.